ql/experimental/convertiblebonds/convertiblebond.hpp

```diff
@@ -188,18 +171,17 @@ namespace QuantLib {
     };
 
 
-    class ConvertibleBond::option::arguments
-        : public OneAssetOption::arguments {
+    class ConvertibleBond::option::arguments : public OneAssetOption::arguments {
       public:
         arguments()
-        : conversionRatio(Null<Real>()), settlementDays(Null<Natural>()),
-          redemption(Null<Real>()) {}
+        : conversionRatio(Null<Real>()), settlementDays(Null<Natural>()), redemption(Null<Real>()) {
+        }
 
         Real conversionRatio;
-        Handle<Quote> creditSpread;
-        DividendSchedule dividends;
-        std::vector<Date> dividendDates;
-        std::vector<Date> callabilityDates;
+        Handle<Quote> creditSpread; //Not being used for calculations in binomailconvertibleengine
+        DividendSchedule dividends; //Not being used for calculations in binomailconvertibleengine
+        std::vector<Date> dividendDates; 
```

Comment:
I would remove these from here as well.  They can be passed from the engine to `DiscretizedConvertible`'s constructor.

Reply:
Done